mautrix_telegram/portal/telegram.py

```diff
@@ -182,7 +182,9 @@ def _parse_telegram_document_meta(evt: Message, file: DBTelegramFile, attrs: Doc
             thumb_loc = None
             thumb_size = None
         file = await util.transfer_file_to_matrix(source.client, intent, document, thumb_loc,
-                                                  is_sticker=attrs.is_sticker)
+                                                  is_sticker=attrs.is_sticker,
+                                                  tgs_convert_type=
+                                                  self.get_config("animated_sticker_target_type"))
```

Comment:
This probably shouldn't be configurable per-room if can affect performance (`get_config` is for per-room configurable stuff).

```suggestion
                                                  config["bridge.animated_sticker_target_type"])
```